pkg/kubelet/dockertools/manager_test.go

```diff
@@ -1822,21 +1824,20 @@ func TestCheckVersionCompatibility(t *testing.T) {
 		// Invalid apiversion
 		{"invalid_api_version", false},
 	}
-	for i := range apiVersion {
-		apiVersion[i]++
+	for i := range intSlice {
```

Comment:
Hahaha, the test is really ugly, I agree, :P. Maybe we can use a very old version and a very new version instead...Such as "0.0.0" and "9999.9999.9999", Hahaha~~WDYT?